integrations/astrovault-adapter/src/dex.rs

```diff
@@ -735,6 +684,124 @@ fn cw_asset_info_to_astrovault(
     }
 }
 
+/// Minimalistic versions of astrovault messages
+/// Currently messages not in use
+#[cfg(feature = "full_integration")]
+mod mini_astrovault {
+    use super::*;
+
+    #[derive(cosmwasm_schema::serde::Deserialize)]
+    #[serde(rename_all = "snake_case", crate = "::cosmwasm_schema::serde")]
+    // Ignoring unknown fields
```

Comment:
Won't this fail to deserialize if too many fields are provided?

Reply:
No, serde ignores unknown fields unless https://serde.rs/container-attrs.html#deny_unknown_fields specified